panacea/consts.go

```diff
@@ -0,0 +1,6 @@
+package panacea
+
+const (
+	CoinType = 371
+	Hrp      = "panacea"
```

Comment:
it is recommeneded to use uppercase `HRP` for abbreviations.

Reply:
I got it! Thanks. fixed in [5a20456](https://github.com/medibloc/panacea-doracle/pull/11/commits/5a2045637c4e4834e5cf21efb76488f73152fd0c)